.github/workflows/frontend_builder.yml

```diff
@@ -0,0 +1,42 @@
+name: Frontend Builder
+
+on:
+  workflow_dispatch:
+  
+  push:
+    branches: [actions]
+    paths-ignore: 
+      - 'build/**'
```

Comment:
This will only run when we push to the specified branch and if there is at least one file changed other than `client/build/**`.